packages/core/src/CloseButton/CloseButton.js

```diff
@@ -6,17 +6,17 @@ import { refPropType } from '../utils'
 import { IconButton } from '../IconButton'
 
 const CloseButton = (props) => (
-  <IconButton {...props} icon={<Close size={props.size} />} />
+  <IconButton {...props} icon={<Close size={props.iconSize} />} />
 )
 
 CloseButton.defaultProps = {
-  size: 24,
+  iconSize: 24,
   title: 'close',
 }
 
 CloseButton.propTypes = {
   onClick: PropTypes.func,
-  size: PropTypes.number,
+  iconSize: PropTypes.number,
```

Comment:
For consistency, I think it'd make sense to keep `size`, but change it to use the same values as `size` on `Button`. Then, we could add `iconSize`, which would control the size of the icon. However, if we remove `size` or change it so it no longer accepts a number, I think that would be a breaking change.